src/parse.c

```diff
@@ -5580,9 +5583,13 @@ static type_t p_record_type_definition(ident_t id)
    type_set_ident(r, id);
    mangle_type(nametab, r);
 
-   do {
-      p_element_declaration(r);
-   } while (peek() == tID);
+   if (peek() == tEND) {
+      require_std(STD_19, "empty record");
+   } else {
+      do {
+         p_element_declaration(r);
+      } while (peek() == tID);
+   }
```

Comment:
```suggestion
   if (peek() == tEND)
      require_std(STD_19, "empty record");
   else {
      do {
         p_element_declaration(r);
      } while (peek() == tID);
   }
```

My normal style is to leave off the braces in if-branches with only one statement and to put the `else` on a new line.